src/screens/IndexScreen.js

```diff
@@ -236,7 +236,7 @@ export const IndexScreen = ({ navigation, route }) => {
               withDate: false
             });
 
-            if (!listItems) return null;
+            if (!listItems?.length && !categories?.length) return null;
```

Comment:
this could be changed to the new `EmptyMessage` from #433 after 2.5.3 merge

Reply:
done with 3197128 and updated empty message text for lists with categories:

![Simulator Screen Shot - iPhone 11 Pro - 2022-04-25 at 14 34 25](https://user-images.githubusercontent.com/1942953/165091307-b9bf7847-8f6b-4280-a56a-420813a31e2f.png)